mm2src/coins/Cargo.toml

```diff
@@ -41,6 +41,7 @@ ethkey = { git = "https://github.com/artemii235/parity-ethereum.git" }
 futures01 = { version = "0.1", package = "futures" }
 # using select macro requires the crate to be named futures, compilation failed with futures03 name
 futures = { version = "0.3", package = "futures", features = ["compat", "async-await"] }
+group = "0.8.0"
```

Comment:
What's the reason to use `group` dependency?

Reply:
It is required to use the method[ to_bytes() ](https://github.com/KomodoPlatform/atomicDEX-API/blob/zcoin_native_mode_support/mm2src/coins/z_coin/z_rpc.rs#L407) in `jubjub::ExtendedPoint`

Reply:
I see, thanks! I have noticed that this dep was in Cargo.lock already, so it's not a new one, no issue with this 🙂